release-notes/README.md

```diff
@@ -5,7 +5,7 @@ The following [.NET releases](../releases.md) are currently supported:
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
 | [.NET 6](6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6/) | [LTS][policies] | [6.0.0 ][6.0.0]  | November 08, 2024 |
-| [.NET 5](5.0/README.md) | [November 10, 2020](https://devblogs.microsoft.com/dotnet/announcing-net-5-0/) | [Current][policies] | [5.0.12][5.0.12] | May 08, 2022 |
+| [.NET 5](5.0/README.md) | [November 10, 2020](https://devblogs.microsoft.com/dotnet/announcing-net-5-0/) | [Maintenance][policies] | [5.0.12][5.0.12] | May 08, 2022 |
```

Comment:
I think we're mixing support level with release types. Maybe we need to start showing both

Reply:
This is explained here. I will re-do the PR
https://github.com/dotnet/core/blob/rbhanda-patch-3/release-policies.md.